components/ui/pathogen-map/use-pathogen-map-mouse.ts

```diff
@@ -36,7 +36,7 @@ export const usePathogenMapMouse = <TPathogenDataPointProperties extends Pathoge
       return;
     }
 
-    setCursor(enteredLayer.cursor);
+    setCursor(enteredLayer?.cursor || baseCursor);
```

Comment:
`enteredLayer` is falsy checked on line 35. Is it possible to remove this change?

Reply:
Silly goose moment